docs/site/tutorials/introducing_x10.ipynb

```diff
@@ -0,0 +1,550 @@
+        "id": "1qad9_yMYf6F",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "eagerTensor1.device"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "HrlMNOinIIy_"
+      },
+      "source": [
+        "If you are running this notebook on a GPU-enabled instance, you should see that hardware reflected in the device description above. The eager runtime does not have support for TPUs, so if you are using one of them as an accelerator you will see the CPU being used as a hardware target."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "eoyLeSQVIIy9"
+      },
+      "source": [
+        "When creating a Tensor, the default eager mode device can be overridden by specifying an alternative. This is how you opt-in to performing calculations using the X10 backend."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "NrRQhQaHaJm9",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "let x10Tensor1 = Tensor([0.0, 1.0, 2.0], on: Device.defaultXLA)\n",
+        "let x10Tensor2 = Tensor([1.5, 2.5, 3.5], on: Device.defaultXLA)\n",
+        "let x10TensorSum = x10Tensor1 + x10Tensor2\n",
+        "x10TensorSum"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "VbqeudQCaqwv",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "x10Tensor1.device"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "mIbIOW0HIIzA"
+      },
+      "source": [
+        "If you're running this in a GPU-enabled instance, you should see that accelerator listed in the X10 tensor's device. Unlike for eager execution, if you are running this in a TPU-enabled instance, you should now see that calculations are using that device. X10 is how you take advantage of TPUs within Swift for TensorFlow.\n",
+        "\n",
+        "The default eager and X10 devices will attempt to use the first accelerator on the system. If you have GPUs attached, the will use the first available GPU. If TPUs are present, X10 will use the first TPU core by default. If no accelerator is found or supported, the default device will fall back to the CPU.\n",
+        "\n",
+        "Beyond the default eager and XLA devices, you can provide specific hardware and backend targets in a Device:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "De59VwJ35SvG",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "// let tpu1 = Device(kind: .TPU, ordinal: 1, backend: .XLA)\n",
```

Comment:
A little strange that this is commented and it's in a code block.

Reply:
As we talked about offline, we want to be able to show an example of specifying a particular device, but if someone runs this on a Colab instance (or Jupyter notebook) that lacks this specific accelerator, we don't want the notebook to fail at this point. They can uncomment this to see how it works on instances that do have this hardware.